flare/components/select.py

```diff
@@ -101,6 +101,19 @@ def build(self, action_row: hikari.api.MessageActionRowBuilder) -> None:
             for option in self.options:
                 if isinstance(option, str):
                     select.add_option(option, option).add_to_menu()
+                elif isinstance(option, hikari.SelectMenuOption):
+                    opt = None
+                    if option.description:
+                        opt = select.add_option(option.label, option.value).set_description(option.description)
+                    else:
+                        opt = select.add_option(option.label, option.value)
```

Comment:
This is what hyper was asking for
```suggestion
                    opt = select.add_option(option.label, option.value)
                    if option.description:
                        opt.set_description(option.description)
```